src/libtiled/gidmapper.cpp

```diff
@@ -38,6 +38,7 @@ using namespace Tiled;
 const int FlippedHorizontallyFlag   = 0x80000000;
 const int FlippedVerticallyFlag     = 0x40000000;
 const int FlippedAntiDiagonallyFlag = 0x20000000;
+const int FlippedLeftHexAntiDiagonallyFlag = 0x10000000;
```

Comment:
Just noting that this `Rotated60Flag` does not need to exist in this file. Since it is only for readability, it can be an alternative getter/setter as `bool Cell::rotated60()`.